detektive/build.gradle.kts

```diff
@@ -5,11 +5,13 @@ plugins {
 
 dependencies {
   compileOnly(Dependencies.detektApi)
+  compileOnly(Dependencies.detektPsiUtils)
   compileOnly(Dependencies.kotlinCompilerEmbeddable)
 
   testImplementation(Dependencies.assertj)
   testImplementation(Dependencies.detektTest)
   testImplementation(Dependencies.detektTestUtils)
+  testImplementation(Dependencies.guice)
```

Comment:
Only needed for my test case which includes a guice annotation (detekt fails to recognise the annotation otherwise).

I've exclude this from the dependency analysis errors as a result (in the root build file)